serverless.yml

```diff
@@ -51,6 +52,15 @@ resources:
           Ref: WebAppS3Bucket
         PolicyDocument:
           Statement:
+            - Sid: "AllowPublicAccess"
```

Comment:
This policy is unnecessary. Also you can't just clone and run cloudfront deploy with it.